### DISCUSSION TITLE
Is this still actively Maintained?

### DISCUSSION THREAD

**Discussion by klauss194** [marp-team/marp#401] [Q&A] [answered]
Dear Friends,

I love the tool.
I would need to ask if this is still actively maintained or slowly fading. 
As well as it feels a bit that the documentation seems scattered. 
(a simple observation is: where can one find instructions for images - the main website lacks a docs link)

Kind regards

**Comment by yhatt** [comment, marked as answer]
No worry! The ecosystem is actively maintained.

I'm also still working on each Marp projects. In fact, this weekend is going to update [Marp Core](https://github.com/marp-team/marp-core), [CLI](https://github.com/marp-team/marp-cli), and [VS Code extension](https://github.com/marp-team/marp-vscode). If these are not maintained, I would not be here to respond actively to Marp users.

---

> As well as it feels a bit that the documentation seems scattered.

We do not regard this as a problem. We actually have separated scope by every projects, to suppress requirement about extra knowledgements for development (such as depending projects) and be easy to contribute into each projects.

For example, developer and contributor who just only want to use [Marpit framework](https://marpit.marp.app/) is not need full documentation about Marp. For [CLI](https://github.com/marp-team/marp-cli) expert, there is no meaning to describe about VS Code extension. So we have essentially different documentations in each scoped tools.

If you want to know the whole of Marp ecosystem, this architecture may feel a bit scattered (as you described). However, it's a result of working on each Marp projects to keep long-live and high flexibillity.

I have seen many other Markdown presentation projects that received many requests and much anticipation, but ended up being stopped maintainance (e.g. [classic Marp app](https://yhatt.github.io/marp/), [mdx-deck](https://github.com/jxnblk/mdx-deck/issues/765)). Separating all and keeping scope of each projects minimum is actually for preventing to become unmaintained projects.

**Reply by klauss194** [reply]
Hey,

Thank you so much for this news.
Completely understandable. 
I will start looking around for contributing!

**Reply by sbibauw** [reply]
@klauss194 I can confirm @yhatt is super active in maintaining Marp: both answering fast on these forums and releasing new versions of Marp components very frequently (±every month).

Regarding your question about image manipulation: https://marpit.marp.app/image-syntax
I agree with you. Even after using Marp for two years, I find the division between _marp, marp-core, marpit, marp-vscode_ confusing. But the Marpit documentation website mentioned above is quite effective and clear in my opinion.

**Comment by klauss194** [comment]
@sbibauw 

Is there any information in regards Scaling? Fitting ? I can see that past a certain point the content is not visible anymore however I am willing to scale down the content to fit.

**Reply by sbibauw** [reply]
Yes: https://marpit.marp.app/image-syntax?id=background-size
At least for background images: `![bg fit](image.jpg)` or `![bg contain](image.jpg)` or `![bg 150%](image.jpg)`

If for inline images, I mostly use definitions in height to control size and alignment, e.g.:
    `![h:200](image1.jpg) ![h:200](image2.jpg)`
This way both images are perfectly aligned on the same line.
